runtime/src/app/router/index.ts

```diff
@@ -19,11 +19,11 @@ export function set_cid(n: number) {
 	cid = n;
 }
 
-const _history = typeof history !== 'undefined' ? history : {
-	pushState: (state: any, title: string, href: string) => {},
-	replaceState: (state: any, title: string, href: string) => {},
-	scrollRestoration: ''
-};
+const _history: History = typeof history !== 'undefined' ? history : {
+	pushState: () => {},
+	replaceState: () => {},
+	scrollRestoration: 'auto'
```

Comment:
Not directly. It's from adding types. This object literal wasn't even a `Partial<History>` because `scrollRestauration` is not allowed to have empty string as value, so I set it to `auto` which is the browser default.